swagger_server/__main__.py

```diff
@@ -18,15 +19,33 @@
 # ROUTES!
 # ------------------------------------------------------------------------------
 
+
 @app.route('/expression', methods=['POST'])
 def post_expression_view():
-    return add_expression()
+    try:
+        return add_expression(), 200
+    except ValidationError as exc:
+        return jsonify_validation_error(exc)
+
+
+@app.route('/expression/<int:expression_id>', methods=['DELETE'])
+def delete_expression_view(expression_id):
+    try:
+        return delete_expression(expression_id)
+    except ValidationError as exc:
+        return jsonify_validation_error(exc)
+
 
 @app.route('/expression/<int:expression_id>', methods=['GET'])
 def get_expression_view(expression_id):
     return get_expression(expression_id)
 
 
+@app.route('/expression/<int:expression_id>', methods=['PUT'])
+def put_expression_view(expression_id):
+    return put_expression(expression_id)
```

Comment:
I think `put_expression` should take both `expression_id` and the payload (i.e. request.get_json)